### PR TITLE
add XYZ Space as APIFormat to OmvDataSource

### PR DESCRIPTION
**Pull request by dbabbs**
Add an XYZ Space as an `APIFormat` to be used within `OmvDataSource`. This enables a developer to visualize data from an XYZ Space, using the `.mvt` endpoint from XYZ. Example endpoint: 

```
https://xyz.api.here.com/hub/spaces/your-space-id/tile/web/{z}_{x}_{y}.mvt?access_token=your-access-token
```

Example of adding data from XYZ Space to map:
```
const xyzDataSource = new OmvDataSource({
   baseUrl: "https://xyz.api.here.com/hub/spaces/2vDrakce/tile/web",
   apiFormat: APIFormat.XYZSpace,
   styleSetName: "geojson",
   maxZoomLevel: 17,
   authenticationCode: "your-access-token"
});
map.addDataSource(xyzDataSource);
```

### CODE REVIEW

File: @here/harp-omv-datasource/lib/OmvRestClient.ts [hunk 1]
```diff
@@ -301,6 +317,8 @@ export class OmvRestClient implements DataProvider {
             case APIFormat.MapboxV4:
             case APIFormat.XYZOMV:
             case APIFormat.XYZMVT:
+            case APIFormat.XYZSpace:
+               return AuthenticationTypeAccessToken;
```
Comment (ninok) [inline @here/harp-omv-datasource/lib/OmvRestClient.ts]: This additional return is not needed. The case will just fall through to line 323

File: @here/harp-omv-datasource/lib/OmvRestClient.ts [hunk 2]
```diff
@@ -362,6 +383,9 @@ export class OmvRestClient implements DataProvider {
             case APIFormat.XYZJson:
                 path += ".json";
                 break;
+            case APIFormat.XYZSpace:
+               path += ".mvt";
```
Comment (ninok) [inline @here/harp-omv-datasource/lib/OmvRestClient.ts]: maybe you can set the whole path here (including the .mvt) and remove the if above

File: @here/harp-omv-datasource/lib/OmvRestClient.ts [hunk 3]
```diff
@@ -348,6 +366,9 @@ export class OmvRestClient implements DataProvider {
      */
     private dataUrl(tileKey: TileKey): string {
         let path = `/${tileKey.level}/${tileKey.column}/${tileKey.row}`;
+        if (this.params.apiFormat === APIFormat.XYZSpace) {
+           path = `/${tileKey.level}_${tileKey.column}_${tileKey.row}`;
+        }
```
Comment (Astrak) [inline @here/harp-omv-datasource/lib/OmvRestClient.ts]: What about 

```typescript
const path = 
    [`/${tileKey.level}`,tileKey.column,tileKey.row]
    .join(
        this.params.apiFormat === APIFormat.XYZSpace ? 
            "_" 
            : "/"
    );
```

Comment (dbabbs) [reply @here/harp-omv-datasource/lib/OmvRestClient.ts]: I implemented this

File: @here/harp-omv-datasource/lib/OmvRestClient.ts (hunk 2 above)
Comment (dbabbs) [reply @here/harp-omv-datasource/lib/OmvRestClient.ts]: I went with @Astrak 's approach

File: @here/harp-omv-datasource/lib/OmvRestClient.ts (hunk 1 above)
Comment (dbabbs) [reply @here/harp-omv-datasource/lib/OmvRestClient.ts]: Done

**Comment by ninok** [comment]
@heremaps-bot retest this please

**Pull request closed without merging**